perception/navigator_vision/nodes/pinger_finder.py

```diff
@@ -0,0 +1,170 @@
+#!/usr/bin/env python
```

Comment:
Let's add this the perception.launch file. Also, is there anything else related to this mission that needs to be launched with master_launch? Is it just the driver and this?